wellcometrust/corporate-components
# #7929 feature: adds MissionStatement component

# Description

As part of the UI updates taking place on the .org homepage we are adding a MissionStatement component which does not currently exist.

It could be argued that this component should live in corporate-react. I have chosen to build it within corporate-components because initially I thought that this component would also be used on the Drupal 7 homepage. 

I am open to suggestions as to whether this component should live in corporate-react, and wouldn't mind closing this PR and re-opening it on corporate-react if that is consensus.

See /wellcometrust/corporate/issues/7929 for further information.

Zeplin: https://app.zeplin.io/project/5b4e06b48ae4580d4871178a/screen/5f803212f498fd5904bab2f5

# To test

1. Pull the branch
2. `npm run storybook`
3. Open "QuickLinks" from left-hand sidebar
4. Inspect component
5. Check against designs in Zeplin https://app.zeplin.io/project/5b4e06b48ae4580d4871178a/screen/5f803212f498fd5904bab2f5
